### PR TITLE
Feature/meetings

### PR DESCRIPTION
**Pull request by DutcherNL**
A new feature to insert meetings into squire. This will be implemented for campaign planning

Still to do:
- [x] Redact is_public field (it is replaced with a choicefield)
- [x] Meeting calendar feed
- [x] Testcases

### CODE REVIEW

File: activity_calendar/constants.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
+##############################################################################
+##                                                                          ##
+##  This file contains constant values that are used throughout the module  ##
+##  To prevent circular loops these are defined here                        ##
+##                                                                          ##
+##############################################################################
+
+
+ACTIVITY_PUBLIC = "PUBLIC"
```
Comment (EricTRL) [inline activity_calendar/constants.py:9]: Consider making this an Enum?

**Review by EricTRL — CHANGES_REQUESTED** [review]

**Review by EricTRL — APPROVED** [review]

**Review by LenaWil — CHANGES_REQUESTED** [review]
Friendly request to at least run the formatter on the files you added

It’s possible to do that offline.

If you really don’t want to or think the formatter is too strict, that’s okay too, but then we should talk about improving that.

https://black.readthedocs.io/en/stable/the_black_code_style/current_style.html

**Comment by DutcherNL** [comment]
> Friendly request to at least run the formatter on the files you added
Is there a way to run them all as a single commit. As you can see it has a lot of suggestions and I rather not gain RSI from formatting xD

**Comment by LenaWil** [comment]
> 
> Is there a way to run them all as a single commit. As you can see it has a lot of suggestions and I rather not gain RSI from formatting xD
XD

Which IDE are you using?

**Review by LenaWil — COMMENTED** [review]
I sent #314, otherwise fine with me.

**Review by LenaWil — APPROVED** [review]

**Pull request merged**